dev/devicelab/lib/framework/browser.dart

```diff
@@ -422,18 +422,6 @@ Duration _computeAverageDuration(List<BlinkTraceEvent> events) {
 /// See also:
 ///  * https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview
 class BlinkTraceEvent {
-  BlinkTraceEvent._({
-    required this.args,
-    required this.cat,
-    required this.name,
-    required this.ph,
-    this.pid,
-    this.tid,
-    this.ts,
-    this.tts,
-    this.tdur,
-  });
```

Comment:
This constructor was only used in `BlinkTraceEvent.fromJson`, so it's no longer needed.